CustomDimensions.php

```diff
@@ -267,11 +267,12 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = 'CustomDimensions_CustomDimensionId';
     }
 
-    public function addConversionInformation(&$conversion, $visitInformation, Tracker\Request $request)
+    public function addConversionInformation(&$conversion, $visitInformation, Tracker\Request $request, $action = null)
     {
         $dimensions = CustomDimensionsRequestProcessor::getCachedCustomDimensions($request);
+        $actionDimensions = is_null($action) ? CustomDimensionsRequestProcessor::getCustomDimensionsInScope(CustomDimensions::SCOPE_ACTION, $request) : $action->getCustomFields();
```

Comment:
This works. We need to require in plugin.json a higher Piwik version though otherwise it will fail when using the plugin eg with Piwik 3.0.1 where the method `getCustomFields()` does not exist. Can you update `plugin.json` the required Piwik version to `>=3.0.2` (I think it is something like `>=3.0.0-dev` or so currently)